lib/resty/lmdb/prefix.lua

```diff
@@ -55,6 +47,14 @@ function _M.page(start, prefix, db, page_size)
     ops[0].dbi = dbi
 
 ::again::
+    ops[0].opcode = C.NGX_LMDB_OP_PREFIX
+    ops[0].key.data = start
+    ops[0].key.len = #start
```

Comment:
A small suggestion: cache #start and #prefix  before `::again::`.

Reply:
Unlike array, the `#` operation on strings are an `O(1)` operation, it should not be necessary as this is nowhere close to a hot path.